app/screens/edit_profile/edit_profile.js

```diff
@@ -436,6 +457,11 @@ export default class EditProfile extends PureComponent {
         this.scrollView = ref;
     };
 
+    componentDidMount() {
+        const {id, last_picture_update: lastPictureUpdate} = this.props.currentUser;
+        this.getProfileImageUri(id, lastPictureUpdate);
```

Comment:
What is the purpose of this? Don’t you already have the image uri?

Reply:
I don't have the image URI from what i can see in the currentUser object.

Reply:
Yeah this just returns the path, I thought it was making a request, sorry for the confusion